data-types.md

```diff
@@ -25,12 +25,18 @@ Type | Description | Example
 
 CockroachDB supports explicit type conversions using the following methods:
 
-- `<value>::<data type>`, or its equivalent longer form `CAST(<value> AS <data type>)`, which converts an arbitrary expression of one built-in type to another (this is also known as "casting"). For example:
-`NOW()::DECIMAL`, `VARIANCE(a+2)::INT`.
+- `<value>::<data type>`, or its equivalent longer form `CAST(<value> AS <data type>)`, which converts an arbitrary expression of one built-in type to another (this is also known as type coercion or "casting"). For example:
+  `NOW()::DECIMAL`, `VARIANCE(a+2)::INT`.
+
+  **Note that in many cases it is advisable to use an
```

Comment:
Nice tip! 2 suggestions:

1. Instead of bolding the whole thing, let's format it as an actual tip. Here's how (plus some suggested copy edits):

```
{{site.data.alerts.callout_success}}
In many cases, it is recommended to use a [type annotation](sql-expressions.html#explicitly-typed-expression) instead of a coercion to create a value of an arbitrary type.
{{site.data.alerts.end}}
```

2. Should we add a bit of explanation about why type annotation is better in most cases?